packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -42,14 +42,12 @@ export class MenuItem extends FASTElement {
     @attr({ attribute: "expanded" })
     public expanded: boolean;
     private expandedChanged(oldValue: boolean): void {
-        const submenu: Element | undefined = this.getSubmenu();
         if (this.$fastController.isConnected) {
-            const submenu: Element | undefined = this.getSubmenu();
-            if (submenu === undefined) {
+            if (this.submenu === undefined) {
                 return;
             }
             if (this.expanded === false) {
-                (submenu as Menu).collapseExpandedItem();
+                (this.submenu as Menu).collapseExpandedItem();
             } else {
                 this.currentDirection = getDirection(this);
```

Comment:
I haven't used getDirection so I am guessing at the purpose here. Is it to inherit direction from the host menu component? From a performance standpoint it makes sense to only set it if the submenu is open. But with the indicator having RTL specific styles, would this need to be called in connectedCallback now?

Reply:
No, we get is specifically to populate the dir attribute of the anchored region when it is created.  All the css layout stuff just works.